### ISSUE TITLE
"bias_attr" not used in batch_norm operator in Fluid

### ISSUE THREAD

**Issue by sidgoyal78** [PaddlePaddle/Paddle#7182]
I was adding the documentation for batch norm layer as part of #6526 . It seems that bias_attr is not used in the implementation (even though it is part of the argument list). 

Maybe instead of `param_attr` for the bias, should we use `bias_attr`?
(https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/layers/nn.py)

**Comment by sidgoyal78** [comment]
@jacquesqiao : I think bias_attr will be part of `**locals()`, but can you help me understand what is the use of `bias_attr` in this operator? (As it doesn't seem to come in the equations at all). Thanks.
